cloud/aws/stress/stress.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# The caller should download the static-tests and extract them into the
+# local directory.
+
+set -x
+
+LOG_DIR="logs"
+NUM_PROCS=4
+MAX_RUNS=0
+MAX_TIME=15m
+MAX_FAILS=1
+
+STRESS_FLAGS="-p ${NUM_PROCS} -maxruns ${MAX_RUNS} -maxtime ${MAX_TIME} -maxfails ${MAX_FAILS} -stderr"
```

Comment:
Actually, you could remove the `-p ${NUM_PROCS}` and just let stress default to the number of processors.